src/engraving/infrastructure/io/mscreader.cpp

```diff
@@ -28,6 +28,8 @@
 #include "io/fileinfo.h"
 #include "serialization/zipreader.h"
 #include "serialization/xmlstreamreader.h"
+#include "engraving/engravingerrors.h"
+#include "libmscore/masterscore.h"

```

Comment:
you'll need something like this

```
namespace mu::engraving {
    static inline bool operator!=(const char* a, const AsciiStringView b)  { 
        return ::operator!=(a, b); 
    }
}
```
to avoid the name hiding problem with operator!= here.